modules/plotting/include/modules/plotting/datastructures/axissettings.h

```diff
@@ -57,6 +57,8 @@ class IVW_MODULE_PLOTTING_API AxisSettings {
     virtual bool getFlipped() const = 0;
     virtual vec4 getColor() const = 0;
     virtual float getWidth() const = 0;
+    // scaling factor affecting tick lengths and offsets of axis caption and labels
+    virtual float getScalingFactor() const = 0;
```

Comment:
Factor is redundant. Could name it scaling, transform, or offsetScaling. OffsetScaling seem to be used in later.
```suggestion
    virtual float getOffsetScaling() const = 0;
```

Reply:
Since it affects more than the offset, according to the comment, I think getScalingFactor or getScaleFactor is appropriate